Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/ConversationFragment.kt

```diff
@@ -30,35 +32,48 @@ import com.example.compose.jetchat.MainViewModel
 import com.example.compose.jetchat.R
 import com.example.compose.jetchat.data.exampleUiState
 import com.example.compose.jetchat.theme.JetchatTheme
+import dev.chrisbanes.accompanist.insets.AmbientWindowInsets
+import dev.chrisbanes.accompanist.insets.ExperimentalAnimatedInsets
+import dev.chrisbanes.accompanist.insets.ViewWindowInsetObserver
 
 class ConversationFragment : Fragment() {
 
     private val activityViewModel: MainViewModel by activityViewModels()
 
+    @OptIn(ExperimentalAnimatedInsets::class) // Opt-in to experiment animated insets support
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View {
-        return ComposeView(context = requireContext()).apply {
-            setContent {
-                Providers(AmbientBackPressedDispatcher provides requireActivity()) {
-                    JetchatTheme {
-                        ConversationContent(
-                            uiState = exampleUiState,
-                            navigateToProfile = { user ->
-                                // Click callback
-                                val bundle = bundleOf("userId" to user)
-                                findNavController().navigate(
-                                    R.id.nav_profile,
-                                    bundle
-                                )
-                            },
-                            onNavIconPressed = {
-                                activityViewModel.openDrawer()
-                            }
-                        )
-                    }
+    ): View = ComposeView(inflater.context).apply {
+        layoutParams = LayoutParams(MATCH_PARENT, MATCH_PARENT)
+
+        // Create an ViewWindowInsetObserver using this view, and call start() to
+        // start listening now. The WindowInsets instance is returned to us, allowing us to
+        // provide it to AmbientWindowInsets in our content below.
+        val windowInsets = ViewWindowInsetObserver(this)
+            .start(windowInsetsAnimationsEnabled = true)
```

Comment:
Since this is a new API, do you want to explain why the animations are enabled here and not in the other instance?